halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/AccountController.java

```diff
@@ -99,7 +99,7 @@
     builder.setFieldOptionsResponse(() -> accountService.getAccountOptions(deploymentName, providerName, accountName, fieldName));
     builder.setSeverity(severity);
 
-    return TaskRepository.submitTask(builder::build);
+    return DaemonTaskHandler.submitTask(builder::build, "Get " + fieldName + " options");
```

Comment:
Yeah, GET requests can't have bodies, and both of these options endpoints you commented on need to supply state to stage for checking what options are available.